core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
           />
         </Card>
       </div>
-      <br />
-      <Heading size="m">Examples</Heading>
-      <Paragraph>Here are some representative examples of these classes:</Paragraph>
-      &nbsp;
-      <div className="pl-10 h-25 bg-light w-50">
-        <span className="border bg-secondary-lightest p-8 d-inline-block mr-6">border</span>
-        <span className="border-top bg-secondary-lightest p-8 d-inline-block mr-6">top</span>
-        <span className="border-right bg-secondary-lightest p-8 d-inline-block mr-6">right</span>
-        <span className="border-bottom bg-secondary-lightest p-8 d-inline-block mr-6">bottom</span>
-        <span className="border-left bg-secondary-lightest p-8 d-inline-block">left</span>
+
+      {/* Examples section */}
+      <Heading size="m" className="mt-8">
+        Examples
+      </Heading>
+      <Text>Here are examples of these border utilities in action:</Text>
+
+      {/* Border examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Placement
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="border bg-secondary-lightest p-8 d-inline-block mr-6">
+            border
+          </Text>
+          <Text appearance="default" className="border-top bg-secondary-lightest p-8 d-inline-block mr-6">
+            top
+          </Text>
+          <Text appearance="default" className="border-right bg-secondary-lightest p-8 d-inline-block mr-6">
+            right
+          </Text>
+          <Text appearance="default" className="border-bottom bg-secondary-lightest p-8 d-inline-block mr-6">
+            bottom
+          </Text>
+          <Text appearance="default" className="border-left bg-secondary-lightest p-8 d-inline-block">
+            left
+          </Text>
```

Comment:
Applying border classes directly to typography components like Text is generally not appropriate. Borders are better suited for layout or wrapper components that handle visual structure rather than text styling.